src/wallet/rpcwallet.cpp

```diff
@@ -3169,6 +3169,76 @@ UniValue generate(const JSONRPCRequest& request)
     return generateBlocks(coinbase_script, num_generate, max_tries, true);
 }
 
+UniValue rescanblockchain(const JSONRPCRequest& request)
+{
+    CWallet * const pwallet = GetWalletForJSONRPCRequest(request);
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() > 2) {
+        throw std::runtime_error(
+            "rescanblockchain (\"start_height\") (\"stop_height\")\n"
+            "\nRescan the local blockchain for wallet related transactions.\n"
+            "\nArguments:\n"
+            "1. \"start_height\"    (number, optional) block height where the rescan should start\n"
+            "2. \"stop_height\"     (number, optional) block height where the rescan should stop\n"
+            "\nResult:\n"
+            "{\n"
+            "  \"start_height\"     (number) The block height where the rescan has started. If omitted, rescan starts from the genesis block.\n"
+            "  \"stop_height\"      (number) The height of the last rescanned block. If omitted, rescan stops at the chain tip.\n"
```

Comment:
tense error s/stops/stopped

Reply:
This is not accurate AFAICT. The block at stop_height is not scanned?

Reply:
AFAIK the block defined with `stop_height` will be scanned.